python/ray/serve/_private/replica.py

```diff
@@ -855,38 +846,6 @@ async def call_user_method_with_grpc_unary_stream(
                     f"function, but '{user_method.__name__}' is not."
                 )
 
-    async def call_user_method_grpc_unary(
-        self, request_metadata: RequestMetadata, request: gRPCRequest
-    ) -> Tuple[RayServegRPCContext, bytes]:
-        """Call a user method that is *not* expected to be a generator.
-
-        Deserializes gRPC request into protobuf object and pass into replica's runner
-        method. Returns a serialized protobuf bytes from the replica.
-        """
-        async with self.wrap_user_method_call(request_metadata):
-            user_request = pickle.loads(request.grpc_user_request)
```

Comment:
Mostly LGTM and thanks for the refactor! I think this pickle loads got left out for gRPC call 🙃

Reply:
oops :)